vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -690,6 +690,18 @@ func (l *terminationLoggingListener) Accept() (net.Conn, error) {
 		return nil, err
 	}
 
+	// ignore connections to local IP. Those clients better know what they are doing.
+	host, _, splitErr := net.SplitHostPort(c.LocalAddr().String())
+	if splitErr != nil {
+		return c, err
+	}
+	if ip := net.ParseIP(host); ip == nil {
+		return c, err
+	} else if ip.IsLoopback() {
+		klog.V(4).Infof("Accepted new local connection from %s very late in the graceful termination process (more than 80%% has passed).", c.RemoteAddr().String())
```

Comment:
how do you know it's late? more than 80%?

Reply:
oh, good point. I first only wanted to ignore those, but decided to log with V(4).

Fixed now.